src/Pos/Util/BackupPhrase.hs

```diff
@@ -2,28 +2,28 @@
 
 module Pos.Util.BackupPhrase
        ( BackupPhrase
-       , mkBackupPhrase
+       , mkBackupPhrase12
+       , mkBackupPhrase9
        , backupPhraseWordsNum
        , bpToList
        , toSeed
        , keysFromPhrase
        , safeKeysFromPhrase
        ) where
 
-import           Crypto.Hash         (SHA3_256)
-import           Data.Char           (isAlpha, isSpace)
-import           Data.List           (span, (!!))
-import           Data.Maybe          (maybeToList)
 import qualified Data.Text           as T
 import           Data.Text.Buildable (Buildable (..))
-import           Prelude             (readsPrec, show)
-import           Universum           hiding (show)
+import qualified Prelude
```

Comment:
This is needed to implement https://github.com/input-output-hk/cardano-sl/pull/176/files#diff-f70d7ade7a372136e0bc2ea8ab665decR50 . See https://serokell.slack.com/archives/C0EAY0GLR/p1489751995832491